src/Application/Regulation/Command/Location/SaveLocationCommandHandler.php

```diff
@@ -109,18 +122,21 @@ private function computeLaneGeometry(SaveLocationCommand $command): ?string
         );
     }
 
-    private function computeDepartmentalRoadGeometry(SaveLocationCommand $command): ?string
+    private function computeRoadSectionGeometry(SaveLocationCommand $command): string
     {
-        if ($command->departmentalRoadGeometry) {
-            return $command->departmentalRoadGeometry;
-        }
-
-        $departmentalRoadNumbers = $this->roadGeocoder->findDepartmentalRoads($command->roadNumber, $command->administrator);
-        if (!$departmentalRoadNumbers) {
-            throw new GeocodingFailureException(sprintf('could not retrieve geometry for roadNumber="%s", administrator="%s"', $command->roadNumber, $command->administrator));
-        }
-
-        return $departmentalRoadNumbers[0]['geometry'];
+        $fullDepartmentalRoadGeometry = $this->roadGeocoder->computeRoad($command->roadNumber, $command->administrator);
+
+        return $this->roadSectionMaker->computeSection(
+            $fullDepartmentalRoadGeometry,
+            $command->administrator,
+            $command->roadNumber,
+            $command->fromPointNumber,
+            $command->fromSide,
+            $command->fromAbscissa,
+            $command->toPointNumber,
+            $command->toSide,
+            $command->toAbscissa,
```

Comment:
Ce que je voulais dire dans https://github.com/MTES-MCT/dialog/pull/693#discussion_r1557509744 ça aurait été de faire ça...

```suggestion
            $command->fromAbscissa ?? 0,
            $command->toPointNumber,
            $command->toSide,
            $command->toAbscissa ?? 0,
```

Et d'exiger un `int` (sans valeur par défaut) dans les fonctions en aval